petclinic.yaml

```diff
@@ -8,43 +8,41 @@ projects:
       location: 'git@github.com:spring-projects/spring-petclinic.git'
 tools:
   - name: mvn-stack
-    definition:
-      dockerImage:
-        image: maven:3.5.4-jdk-8
-        entryPoint: sleep infinity
-        persistentVolumes:
-          - name: maven-repo
-            containerPath: /root/.m2
-        services:
-          - name: spring-boot
-            port: 8080
+    type: dockerImage
+    image: maven:3.5.4-jdk-8
+    entryPoint: sleep infinity
+    persistentVolumes:
```

Comment:
Good idea. It makes me think that for the dockerImage definition (and other 3rd party objects) we should try to reuse existing API as much as possible instead of reinventing ours.